src/schematics/nightwatch/schema.json

```diff
@@ -36,6 +36,11 @@
       "default": "firefox",
       "x-prompt": "Which browser do you want to run the test on? (default: firefox)"
     },
+    "componentTesting": {
+      "default": false,
+      "type": "boolean",
+      "x-prompt": "Would you like to enable component testing?"
```

Comment:
this is not needed as it shows `y/N` in the prompt.

Reply:
This will let users know what would happen if they use `enter`, whereas schematics will use the default value when the user provides no options. Moreover, this is how the above questions are structured.

Reply:
This is the angular's own question: 
<img width="941" alt="Screenshot 2023-04-03 at 6 51 01 PM" src="https://user-images.githubusercontent.com/2018070/229522146-0c7c024e-928e-49b3-8cf6-0e757d0deb99.png">

This is the one I added:
<img width="504" alt="Screenshot 2023-04-03 at 6 52 07 PM" src="https://user-images.githubusercontent.com/2018070/229522364-2e723a3f-1d99-4c27-8db4-1fced7c81292.png">

Just trying to be consistent. 🙂

Reply:
Ah! can you update the rest of the questions too? This way, we will be consistent across the questions

Reply:
Done